docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md

```diff
@@ -0,0 +1,37 @@
+# What is Nebula Operator
+
+## Concept of Nebula Operator
+
+Nebula Operator is a tool to automate the deployment, and operation and maintenance of [Nebula Graph](https://github.com/vesoft-inc/nebula) clusters on [Kubernetes](https://kubernetes.io). Building upon the excellent scalability mechanism of Kubernetes, Nebula Graph introduced its operation and maintenance knowledge into the Kubernetes system, which makes Nebula Graph a real [cloud-native graph database](https://www.nebula-cloud.io/).
+
+## How it works
+
+For resource types that do not exist within Kubernetes，you can register them by adding custom API objects. The common way is to use the [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions).
+
+Nebula Operator abstracts the deployment management of Nebula Graph clusters as a CRD. By combining multiple built-in API objects including StatefulSet, Service, and ConfigMap, the routine management and maintenance of a Nebula Graph cluster are coded as a control loop in the Kubernetes system. When a CR instance is submitted, Nebula Operator drives database clusters to the final state according to the control process.
+
+## Functionalities of Nebula Operator
+
+The following features are already available in Nebula Operator:
+
+- **Deploy and uninstall clusters**：Nebula Operator simplifies the process of deploying and uninstalling clusters for users. Nebula Operator allows you to quickly create, update, or delete a Nebula Graph cluster by simply providing the corresponding CR file. For more information, see [Deploy Nebula Graph Clusters with Kubectl](3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md) or [Deploy Nebula Graph Clusters with Helm](3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md).
+  
+- **Scale clusters**：Nebula Operator calls Nebula Graph's native scaling interfaces in a control loop to implement the scaling logic. You can simply perform scaling operations with YAML configurations and ensure the stability of data. For more information, see [Scale clusters with Kubectl](3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md#_3) or [Scale clusters with Helm](3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md#_2).
+  
+- **Self-Healing**：Nebula Operator calls the interface provided by Nebula Graph clusters to dynamically sense cluster service status. Once an exception is detected, Nebula Operator performs fault tolerance. For more information, see [Self-Healing](5.operator-failover.md).
```

Comment:
The paragraph above said that "Nebula Graph's native scaling interfaces", but here you used "the interface provided by Nebula Graph clusters". I guess the latter sentence should use the plural form as well?